floyd/cli/data.py

```diff
@@ -141,6 +141,66 @@ def clone(id):
                               delete_after_untar=True)
 
 
+@click.command()
+@click.argument('id', nargs=1)
+def listfiles(id):
```

Comment:
`id` is a builtin keyword in python, so the best practice is to avoid naming variable using it. In this case, we should use `name` instead since we have already moved away from the human unfriendly random id. The client only supports it for backwards compatibility reason. We should encourage users to use the new human friendly name whenever we can :)